packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -473,6 +475,10 @@ class ScrollableBaseNative extends Component {
 		this.lastFocusedItem = null;
 		this.lastScrollPositionOnFocus = null;
 		this.isWheeling = false;
+		if (this.isVoiceControl) {
+			this.isVoiceControl = false;
+			this.updateFocusAfterVoiceControl();
```

Comment:
It looks that the focus will not appear until scroll stops. Is this intended?

Reply:
As @0x64 mentioned, the item should not get focus after stopping scrolling. The item should get focus after creating and before showing it and stopping scrolling. Please check ENYO-4806 issue.